src/components/Disputes/DisputeInfo.js

```diff
@@ -176,6 +109,122 @@ const DisputeInfo = React.memo(function({
   )
 })
 
+function DisputeHeader({ id, dispute }) {
+  const theme = useTheme()
+  const transaction = dispute && dispute.txHash
+
+  return (
+    <div
+      css={`
+        display: flex;
+        margin-bottom: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+        `}
+      >
+        <div
+          css={`
+            background: linear-gradient(
+              233deg,
+              ${theme.accentEnd} -50%,
+              ${theme.accentStart} 91%
+            );
+            border-radius: 50%;
+            padding: 12px;
+            display: inline-block;
+          `}
+        >
+          <img src={iconCourt} alt="" width="39" height="32" />
+        </div>
+        <div
+          css={`
+            margin-left: ${3 * GU}px;
+          `}
+        >
+          <h1
+            css={`
+              display: flex;
+              align-items: center;
+              margin-bottom: ${1 * GU}px;
+              ${textStyle('title3')};
+            `}
+          >
+            <span>Dispute #{id}</span>
+            {dispute && (
+              <DisputeStatus
+                dispute={dispute}
+                css={`
+                  margin: 0 0 0 ${1 * GU}px;
+                `}
+              />
+            )}
+          </h1>
+          {transaction && <TransactionBadge transaction={transaction} />}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function Field({ label, value, isNode }) {
```

Comment:
I notice that `isNode` is used only once: https://github.com/aragon/court-dashboard/pull/89/files#diff-e447e120bfda3c5da3d624e3174f5292R77-R81

But `DisputeRoundPill` is defining its own text styles: https://github.com/aragon/court-dashboard/pull/89/files#diff-44c96f491e62b75470988ff5bf1a3fb9R18-R19

So I think we could drop the `isNode` prop altogether, and always return `value` wrapped in the `textStyle('body2')` element? We could also make it a `div` rather than a span, to accept block elements as well.

Reply:
Ah yes good call 👍